src/components/chat/components/room/room-container.tsx

```diff
@@ -26,6 +27,7 @@ export class RoomContainer extends React.PureComponent<Props, State> {
     this.state = {
       conversation: [],
       message: '',
+      myUserId: '',
```

Comment:
This should not be in the state since the `user` object is received via `props` unless there's a good reason to have it